pkg/commands/pfmerge.ts

```diff
@@ -0,0 +1,13 @@
+import { Command, CommandOptions } from "./command.ts";
+
+/**
+ * @see https://redis.io/commands/pfmerge
+ */
+export class PfMergeCommand extends Command<"OK", "OK"> {
+  constructor(
+    cmd: [destination_key: string, source_key: string, ...keys: string[]],
```

Comment:
I think we made a mistake in this `cmd`.

In the doc it says

```PFMERGE hll3[destination_key] hll1 hll2[number of source_keys]```

I believe we don't need `source_key` this we need `...keys: string[]`. And also allowing array as an input would be nice for source_keys similar to comments above.